### PR TITLE
Added Redux Tutorial

### PR DESCRIPTION
**Pull request by julioalfian**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-id.md [hunk 1]
```diff
@@ -189,7 +189,7 @@
 
 * [Dokumentasi React Bahasa Indonesia](https://id.reactjs.org)
 * [React JS Untuk Pemula](https://masputih.com/2021/05/ebook-gratis-reactjs-untuk-pemula) *(Membutuhkan akun Leanpub atau email yang valid)*
-* [Tutorial React JS Untuk Pemula (React Hooks)](https://mfikri.com/artikel/reactjs-pemula) - Mfikri (HTML)
+* [React Redux Tutorial Untuk Pemula](https://medium.com/codeacademia/tutorial-redux-bagian-i-membuat-todo-list-c26a979d0a1f) - Yudi Krisnandi
```
Comment (shrn01) [inline books/free-programming-books-id.md:192]: Please dont remove old entries while adding new entries

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1310,6 +1310,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
     * [React Tutorial - Part 3](http://www.joshfinnie.com/blog/reactjs-tutorial-part-3/) - Josh Finnie
 * [React with ASP.NET Core Tutorial](https://reactjs.net/getting-started/aspnetcore.html)
 * [React.js Tutorial: Now is Your Time to Try It, Right in Your Browser](https://codegeekz.com/react-js-tutorial/)
+* [Redux Tutorial](https://www.tutorialspoint.com/redux/) - Tutorial Point
```
Comment (shrn01) [inline books/free-programming-books-langs.md:1313]:
```suggestion
* [Redux Tutorial](https://www.tutorialspoint.com/redux/) - Tutorial Point (HTML, PDF)
```

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: books/free-programming-books-langs.md (hunk 2 above)
Comment (julioalfian) [reply books/free-programming-books-langs.md:1313]: okay

File: books/free-programming-books-id.md (hunk 1 above)
Comment (shrn01) [reply books/free-programming-books-id.md:192]: Why was it removed? I am able to access the resource

Comment (julioalfian) [reply books/free-programming-books-id.md:192]: it's not a book, just a tutorial website page

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**